modules/ROOT/pages/native/ds-android-intro.adoc

```diff
@@ -0,0 +1,121 @@
+// tag::excludeDownstream[]
+include::../_partials/attributes.adoc[]
+// end::excludeDownstream[]
+
+
+
+=== Data Sync SDK for Native Android and iOS
```

Comment:
@finp WDYT about linking it somewhere as an alternative Android/IOS community path?